File: CS/XpoTutorial3/MyDataModelCode/Customer.cs
using System;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
namespace XpoTutorial3.DataModel {

    public partial class Customer {
        public Customer(Session session) : base(session) { }
        public override void AfterConstruction() { base.AfterConstruction(); }
    }

}

File: VB/XpoTutorial3/MyDataModelCode/Customer.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Namespace XpoTutorial3.DataModel

	Partial Public Class Customer
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Public Overrides Sub AfterConstruction()
			MyBase.AfterConstruction()
		End Sub
	End Class

End Namespace
